source/B2BApi/HostFactory.cs

```diff
@@ -88,7 +90,15 @@ public static IHost CreateHost(TokenValidationParameters tokenValidationParamete
                         .AddOutgoingMessagesModule(context.Configuration)
                         .AddProcessModule(context.Configuration)
                         .AddMasterDataModule(context.Configuration)
-                        .AddDataAccessUnitOfWorkModule(context.Configuration);
+                        .AddDataAccessUnitOfWorkModule()
+
+                        // Audit log (outbox publisher)
+                        .AddAuditLogOutboxPublisher()
+
+                        // Outbox module and outbox processing
```

Comment:
```suggestion
                        // Outbox module and outbox processing
```

The naming on `AddOutboxModule` and `AddOutboxModule` is suffice for me 💯